gradle/root/coverage.gradle

```diff
@@ -1,43 +1,56 @@
-task mockitoCoverage(type: JacocoReport) {
-
-    allprojects { currentProject ->
-        // These tests do not have a `test` task.
-        if (currentProject.name in ['android']) {
-            return
+allprojects { project ->
+    plugins.withId("java") {
+        project.apply plugin: "jacoco"
+        project.jacoco {
+            toolVersion = '0.8.8'
```

Comment:
Can we switch to [0.8.10](https://github.com/jacoco/jacoco/releases/tag/v0.8.10) already?

Reply:
Or maybe better not, to have more clear path in case of regression.

Reply:
Yep, I didn't update so we can compare the outputs.